src/equations/shallow_water_wet_dry_2d.jl

```diff
@@ -0,0 +1,862 @@
+    return Trixi.flux_fjordholm_etal(u_ll, u_rr, normal_direction,
+                                     ShallowWaterEquations2D(equations.gravity,
+                                                             equations.H0))
+end
+
+"""
+    flux_wintermeyer_etal(u_ll, u_rr, orientation_or_normal_direction,
+                          equations::ShallowWaterEquationsWetDry2D)
+
+Total energy conservative (mathematical entropy for shallow water equations) split form.
+When the bottom topography is nonzero this scheme will be well-balanced when used as a `volume_flux`.
+The `surface_flux` should still use, e.g., [`flux_fjordholm_etal`](@ref).
+
+Further details are available in Theorem 1 of the paper:
+- Niklas Wintermeyer, Andrew R. Winters, Gregor J. Gassner and David A. Kopriva (2017)
+  An entropy stable nodal discontinuous Galerkin method for the two dimensional
+  shallow water equations on unstructured curvilinear meshes with discontinuous bathymetry
+  [DOI: 10.1016/j.jcp.2017.03.036](https://doi.org/10.1016/j.jcp.2017.03.036)
+"""
+@inline function Trixi.flux_wintermeyer_etal(u_ll, u_rr, orientation::Integer,
+                                             equations::ShallowWaterEquationsWetDry2D)
+    return Trixi.flux_wintermeyer_etal(u_ll, u_rr, orientation,
+                                       ShallowWaterEquations2D(equations.gravity,
+                                                               equations.H0))
+end
+
+@inline function Trixi.flux_wintermeyer_etal(u_ll, u_rr,
+                                             normal_direction::AbstractVector,
+                                             equations::ShallowWaterEquationsWetDry2D)
+    return Trixi.flux_wintermeyer_etal(u_ll, u_rr, normal_direction,
+                                       ShallowWaterEquations2D(equations.gravity,
+                                                               equations.H0))
+end
+
+# Specialized `DissipationLocalLaxFriedrichs` to avoid spurious dissipation in the bottom topography
+@inline function (dissipation::Trixi.DissipationLocalLaxFriedrichs)(u_ll, u_rr,
+                                                                    orientation_or_normal_direction,
+                                                                    equations::ShallowWaterEquationsWetDry2D)
+    return (dissipation::Trixi.DissipationLocalLaxFriedrichs)(u_ll, u_rr,
+                                                              orientation_or_normal_direction,
+                                                              ShallowWaterEquations2D(equations.gravity,
+                                                                                      equations.H0))
+end
+
+# Specialized `FluxHLL` to avoid spurious dissipation in the bottom topography
+@inline function (numflux::Trixi.FluxHLL)(u_ll, u_rr, orientation_or_normal_direction,
+                                          equations::ShallowWaterEquationsWetDry2D)
```

Comment:
```suggestion
@inline function (numflux::FluxHLL)(u_ll, u_rr, orientation_or_normal_direction,
                                          equations::ShallowWaterEquationsWetDry2D)
```
To match the way this is done in 1D?

Reply:
Right, this is not necessary anymore and can be removed, the same applies to `DissipationLocalLaxFriedrichs`. I applied these changes in 276990a641d5e020cead93f0520679b12f2a43a3.